react/components/EXPERIMENTAL_Table/typings.d.ts

```diff
@@ -31,33 +31,11 @@ type TableProps = {
   onRowClick?: ({ rowData: unknown }) => void
 }
 
-type BulkedItem = {
-  id: number
-}
-
-type BulkState = {
-  selectedRows?: Array<BulkedItem>
-  allLinesSelected?: boolean
-}
-
-type Bulk = {
-  bulkState?: BulkState
-  hasBulkActions?: boolean
-  hasPrimaryBulkAction?: boolean
-  hasSecondaryBulkActions?: boolean
-  selectAllRows?: () => void
-  deselectAllRows?: () => void
-  selectRow?: (row: BulkedItem) => void
-  setSelectedRows?: (selectedRows: Array<BulkedItem>) => void
-  setAllLinesSelected?: (allLinesSelected: boolean) => void
-  selectAllVisibleRows?: () => void
-}
-
 type TableState = {
   visibleColumns?: Array<Column>
   columns?: Array<Column>
   visibleColumns?: Array<Column>
-  items?: Array<Object>
+  items?: Array<Object> | any
```

Comment:
any? :cry:

Reply:
Yep! For reusability proposes. A Table can receive items of any kind. Since this PR it's already big, I'll do another one that uses generics instead of simply allowing `any`.